### DISCUSSION TITLE
color is not good choice over 7.5 star (can't see)

### DISCUSSION THREAD

**Discussion by Sitoria** [ppy/osu#17920] [Ideas] [answered]
as screenshot see, 7.5 above color is too dark to see star rating, need to change SR color to yellow that osu-web does, or something.
![image](https://user-images.githubusercontent.com/85770829/164585408-390b28f8-16d7-47dc-bf1a-59af108a6ec1.png)

and plus, this is other question, is osu lazer have not patched star rating patch? it have miss match that info and select part.
![image](https://user-images.githubusercontent.com/85770829/164585636-6c3080d9-f245-4658-a0e7-90b24705cdba.png)
![image](https://user-images.githubusercontent.com/85770829/164585795-dc61b921-f3a9-496f-a8b8-c767222a7542.png)

**Comment by peppy** [comment, marked as answer]
Lazer uses potentially a newer calculation method than that provided by the website (which is pretty weird IMO, it should likely be locked together).

The colour issue implies the tooltip needs a design pass to make sure it's always visible.

**Reply by peppy** [reply]
@arflyte do you have an updated design for tooltips? Should we just show the pill inline in the tooltip here?

**Reply by arflyte** [reply]
no updated design for tooltips yet. probably going to be similar to the new song select's diff options. for now, let's just show the inline pills.

**Reply by peppy** [reply]
Should be resolved via https://github.com/ppy/osu/pull/17923

**Reply by Sitoria** [reply]
thanks to apply
